source/api_documentation/network_integration/js_tags/js_revision_parameters.rst

```diff
@@ -0,0 +1,168 @@
+
+.. warning::
+    The provided documentation and JSON response for JS Tag Revision documentation might change in the next release.
+
+.. list-table::
+  :widths: 15 15 50
+  :header-rows: 1
+  :class: parameters
+
+  * - Property
+    - Type
+    - Value
+
+  * - external_revision_id
+    - string (read-only)
+    - The current recent revision id if live or nil if draft.
+
+  * - allow_overflow
+    - boolean (default false)
+    - Reserve one phone number for "overflow" and apply to any additional visitors.
+
+  * - auto_create_local_ringpools
+    - boolean (default false)
+    - Auto create a ring pool with local numbers
+
+  * - auto_create_toll_free_ringpools
+    - boolean (default false)
+    - Auto create a ring pool with toll free numbers
+
+  * - auto_swap
+    - boolean (default false)
+    - Set to true if you want the Invoca JS to automatically detect and replace all numbers on page. This can be useful if you have a large list of phone numbers to manage, or don't know all of the numbers listed across your site. If true, `number_selector` and `number_to_replace` are not needed.
+
+  * - body
+    - string
+    - Response body of the JS Tag to be added to the site
+
+  * - cache_lifetime_days
+    - integer (default 30)
+    - This setting determines how long Invoca identifies a visitor to your landing page as a unique visitor. By default, your Tag's attribution window is set to 30 days.
+
+  * - custom_code
+    - string
+    - The custom code is a scoped function to insert any custom JavaScript to manipulate the options config that will be passed into the Invoca Integration.
+
+  * - **custom_data_fields**
+    - array of objects
+    - Marketing data fields
+
+  * - custom_data_fields[].enabled
+    - boolean
+    - Status of the marketing data field.
+
+  * - custom_data_fields[].partner_name
+    - string
+    - Marketing data field name.
+
+  * - custom_data_fields[].friendly_name
+    - string
+    - Marketing data field friendly name.
+
+  * - custom_data_fields[].attribution_model
+    - string
+    - Marketing data field model type.
+
+  * - custom_data_fields[].data_source_type
+    - string
+    - Data source for the marketing data field.
+
+  * - custom_data_fields[].data_source_name
+    - string
+    - Friendly name for the marketing data field.
+
+  * - default_campaign
+    - object
+    - Default campaign for the all numbers found except for ones in `numbers_to_replace` and `numbers_to_not_replace` configurations.
+
+  * - default_campaign.default_campaign_id
+    - integer
+    - Internal campaign id of the default campaign.
+
+      TBD: SHOULD THIS BE UPDATED TO advertiser_campaign_id_from_network.
```

Comment:
I believe these map to the id_from_network, updating a campaign's id_from_network requires an update to the tag to make sure it continues to map to that campaign.

Reply:
I think this is actually just the internal ID based on a query utilizing it: https://github.com/Invoca/web/blob/18481a657463a8ebf154a9b8c7c36db5cc73af5a/app/models/js_tag/tag.rb#L387